packages/design-system/src/components/textArea/index.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2021 Google LLC
```

Comment:
Not related to this PR but this should be 2023 probably? cc @swissspidy

Reply:
Yes, I noticed that somewhere by the end. I should update all the new headers, but I mostly just copy-paste it from some other file. Can I automate updating for all new files in this PR, @swissspidy?

Reply:
Sure